main.go

```diff
@@ -0,0 +1,108 @@
+package main
+
+import (
+    "strings"
+    "fmt"
+    "go/parser"
+    "go/token"
+    "go/ast"
+    "math/rand"
+    "strconv"
+    "os"
+    "io/ioutil"
+    "regexp"
+)
+
+type FuncTest []string
+
+type FuncTests []FuncTest
+
+type FileTest map[string]FuncTests
+
+func GenerateString (length int) (str string) {
+    n := rand.Intn(length)
+    str = ""
+    for i := 0; i < n; i++ {
+        str += string(rand.Intn(128))
+    }
+    return
+}
+
+func GenerateFuncTests (testedFunc *ast.FuncDecl, funcTestAmt int) (funcTests FuncTests) {
+    funcTests = make([]FuncTest, funcTestAmt)
+    paramTypes := []string{}
+    for _, param := range testedFunc.Type.Params.List {
+        typeName := param.Type.(*ast.Ident).Name
+        for range param.Names {
+            paramTypes = append(paramTypes, typeName)
+        }
+    }
+    paramAmt := len(paramTypes)
+    for i := range funcTests {
+        funcTests[i] = make(FuncTest, paramAmt)
+        for j := range paramTypes {
+            switch paramTypes[j] {
+            case "int":
+                funcTests[i][j] = strconv.Itoa(int(rand.Uint64()))
```

Comment:
Please create a function generateInt and move it to to a file with GenerateString